dd-sdk-android/src/main/kotlin/com/datadog/android/v2/core/internal/SdkCoreRegistry.kt

```diff
@@ -69,6 +69,6 @@ internal class SdkCoreRegistry(
     // endregion
 
     companion object {
-        const val DEFAULT_INSTANCE_NAME = "main"
+        const val DEFAULT_INSTANCE_NAME = "_dd.sdk_core.default"
```

Comment:
do we change it to avoid possible clash with user provided name? This name will be used in the logcat output btw.